db/re/item_db_equip.yml

```diff
@@ -201476,6 +202639,58 @@ Body:
       }
       if (.@r>=11)
          autobonus "{ bonus2 bMagicAtkEle,Ele_Neutral,20; bonus2 bMagicAtkEle,Ele_Holy,20; }",1,10000,BF_WEAPON;
+  - Id: 480278
+    AegisName: Huse_Luk_R_Wing
+    Name: Garuda Wings
+    Type: Armor
+    Weight: 100
+    Defense: 10
+    Slots: 1
+    Locations:
+      Garment: true
+    ArmorLevel: 1
+    EquipLevelMin: 70
+    Refineable: true
+    View: 160
+    Script: |
+      .@r = getrefine();
+      bonus bMdef,15;
+      if (BaseClass == Job_Dancer || BaseClass == Job_Rogue || BaseClass == Job_Monk || BaseClass == Job_Assassin) {
```

Comment:
Shouldn't it be BaseJob ?
```
BaseClass   - The character's 1-1 'normal' job, regardless of Upper value.
              For example, this will return Job_Acolyte for Acolyte, Priest/Monk,
              High Priest/Champion, and Arch Bishop/Sura. If the character has not
              reached a 1-1 class, it will return Job_Novice.
BaseJob     - The character's 'normal' job, regardless of Upper value.
              For example, this will return Job_Acolyte for Acolyte,
              Baby Acolyte, and High Acolyte.
```

Reply:
oh I will change as basejob Thanks